SASS/1-blocks/_nav.scss

```diff
@@ -0,0 +1,52 @@
+header{
+     background:white;
+     text-align:center;
+     position: absolute;
+     z-index: 999;
+    width:100%;
+}
+nav{
+    background: white;
+     width:100%;
+    position: absolute;
+    text-align: left;
+    top: 100%;
+    left: 0;
+     display: none;
+    // transform: scale(1,0);
+    // transform-origin: top;
+    // transition: transform 400ms ease-in-out;
```

Comment:
`REQUIRED`

Consider deleting commented out codes. If you need to review a previous version of your code, use git history.